apps/client/src/features/rundown/event-block/composite/EventBlockChip.module.scss

```diff
@@ -6,14 +6,18 @@
   color: $label-gray;
   padding: 0.125rem 0.5rem;
   border-radius: 2px;
-
+  
   &.over {
     color: $playback-negative;
   }
-
+  
   &.under {
     color: $playback-ahead;
   }
+  
+  &.ontime {
```

Comment:
maybe avoid using this word? maybe we dont need this colour? it could be just the green?

Reply:
green - under time
red - over time
ontime ? blue/orange/purple ? something else? (felt like a nice place to sneak in the logo colour and i makes sense 😉  )

Reply:
I would personally keep it simple

green: no overtime (on time or time ahead)
Orange: overtime 

this is consistent with the colours in other places